db/migrate/20220523204320_create_categories.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCategories < ActiveRecord::Migration[7.0]
+  def change
+    create_table :categories do |t|
+      t.string :icon
+      t.string :name
+      t.references :user, null: false, foreign_key: true
+
```

Comment:
- [ ] Kindly fix the column name in all the user's foreign keys, we should reference the author, according to the ERD, we should have an `author_id` as a user's foreign key column name. 

![schema](https://user-images.githubusercontent.com/53124548/170355704-d9341501-fcfb-4c7b-addd-1ee084b2387a.png)